guides/common/modules/ref_supported-virtualization-infrastructures.adoc

```diff
@@ -0,0 +1,12 @@
+[id="supported-virtualization-infrastructures_{context}"]
+= Supported Virtualization Infrastructures
+
+You can connect {Project} using compute resources to the following virtualization infrastructures:
+
+* KVM (libvirt)
+* {oVirt}
+* VMware
+ifndef::satellite[]
+* Proxmox
+* {KubeVirt}
```

Comment:
IIRC KubeVirt is shipped as a tech preview in Satellite

Reply:
We don't have it in Release Notes nor do we include the chapter about it in Satellite. I'm keeping it as it is for now. If you insist, please file a BVZ for that + Satellite version from which we are supposed to include it as a TechPreview.

Reply:
https://access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html/provisioning_guide/provisioning-virtual-machines-kubevirt_provisioning documents it, so at least since Satellite 6.10 it's been possible.

Reply:
OK, I'll file the BZ :))

Reply:
Oh, sorry! We actually include it in the docs, but it's missing in the RNs.